sonic_data_client/db_client.go

```diff
@@ -1089,7 +1089,7 @@ func dbSingleTableKeySubscribe(c *DbClient, rsd redisSubData, updateChannel chan
 						continue
 					}
 					tblPath.tableKey = subscr.Channel[prefixLen:]
-					err = tableData2Msi(&tblPath, false, nil, &newMsi)
+					err = tableData2Msi(&tblPath, true, nil, &newMsi)
```

Comment:
>true [](http://example.com/codeflow?start=35&length=4)

Seems this is the first time `true` is used for this function. Let's add extensive unit test?

Reply:
To be more clear, could you add unit test to `tableData2Msi` to cover the second argument as `true`?

Reply:
Added UT